test/e2e/adminUI/tests/group003List/uxTestListView.js

```diff
@@ -21,184 +21,145 @@ module.exports = {
 		browser.end();
 	},
 	'List view should allow users to create a new list item': function (browser) {
-		browser.adminUIApp
-			.click('@fieldListsMenu')
-			.click('@nameListSubmenu')
-			.waitForListScreen();
+		browser.adminUIApp.click('@fieldListsMenu').click('@nameListSubmenu').waitForListScreen();
 
-		browser.adminUIListScreen
-			.click('@createFirstItemButton');
+		browser.adminUIListScreen.clickCreateItemButton();
 
-		browser.adminUIApp
-			.waitForInitialFormScreen();
+		browser.adminUIApp.waitForInitialFormScreen();
 
 		browser.adminUIInitialFormScreen.fillFieldInputs({
 			modelTestConfig: NameModelTestConfig,
 			fields: {
-				'name': {value: 'Name Field Test 1'},
-				'fieldA': {firstName: 'First 1', lastName: 'Last 1'},
+				'name': { value: 'Name Field Test 1' },
+				'fieldA': { firstName: 'First 1', lastName: 'Last 1' },
 			}
 		});
 
 		browser.adminUIInitialFormScreen.save();
 
-		browser.adminUIApp
-			.waitForItemScreen();
+		browser.adminUIApp.waitForItemScreen();
 
-		browser.adminUIApp
-			.click('@fieldListsMenu')
-			.click('@nameListSubmenu')
-			.waitForListScreen();
+		browser.adminUIApp.click('@fieldListsMenu').click('@nameListSubmenu').waitForListScreen();
```

Comment:
This should be something like `browser.adminUIApp.openList({section: 'Fields', list: 'Names'})`.
The `openList` function already exists.